last_gas/domain/schedulers/time_scheduler.py

```diff
@@ -66,6 +66,20 @@ async def sad_mondays(bot: Any, channel_id: str) -> None:
     await channel.send(file=discord.File("assets/images/john_kleber_monday.jpeg"))
 
 
+async def ximira_xelo(bot: Any, channel_id: str) -> None:

```

Comment:
looks like we're duplicating code every time we create a schedule, creating functions with the same content, for the future, check this.